src/ensembl/src/content/app/custom-download/components/panel/Panel.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
```

Comment:
This component sits in CustomDownload, but has a very generic name. Should it be CustomDownloadPopup? Or do we expect this popup to reappear in other apps and suspect that it belongs in the shared folder?

Reply:
It might reappear as Andrea mentioned but I'm not sure what else it requires to be put in the shared folder. Also, I don't think it can be called as a popup because of the way it is used in the Preview Download screen.

Reply:
`CustomDownloadInfoBox`?
Or perhaps Andrea can offer some naming suggestions? `Panel` is just far too broad and non-specific.

Reply:
Andrea did mention that this might be re-used within bulk downloads so I am not sure if it we call it something more specific to custom downloads and move it outside later with a different name.

Reply:
Spoke to Andrea regarding this and she suggested to use `CustomDownloadInfoCard`.

Reply:
👌